spec.md

```diff
@@ -1530,10 +1534,11 @@ for x in 1, 2:
 ```
 
 Starlark (like Python 3) does not accept an unparenthesized tuple
-expression as the operand of a list comprehension:
+or lambda expression as the operand of a `for`-clause in a comprehension:
```

Comment:
Python 3 does not accept an unparenthesized tuple in either a `for` or `if` clause, and it does not accept an unparenthesized lambda in a `for` clause.

Reply:
Regarding tuples, Python has never allowed an Expression (unparenthesized tuple) as the operand of an 'if' clause, nor would anyone want such a thing, so we needn't mention it. However, Python2 used to allow an Expression as the operand of 'for', and it's a common and natural thing to want, hence we point out that we follow Python3 in using a Test (tuples must be parenthesized).

By itself, using Test would permit lambda to be the operand of a for clause, but Python3 specifically rejects lambda there, and also in cond expressions (hence the TestNoCond production in the parser implementations).

I think the wording is correct.